<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductButton</class>
 <widget class="QWidget" name="ProductButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>206</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_item_image">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;</string>
   </property>
   <property name="text">
    <string>(Image)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_item_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>204</y>
     <width>206</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK TC Bold</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
color: rgb(75, 75, 75);</string>
   </property>
   <property name="text">
    <string>悟饕排骨飯包</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_item_price">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>252</y>
     <width>206</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
color: rgb(170, 85, 0);</string>
   </property>
   <property name="text">
    <string>NT. 85</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sold_out">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>48</y>
     <width>109</width>
     <height>109</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>(sold out)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
